lib/v2/wechat/feeddd.js

```diff
@@ -51,6 +46,18 @@ module.exports = async (ctx) => {
                     desc: $('#js_content').html(),
                 });
 
+                item.author = $('meta[name=author]').attr('content');
+
+                // another way to get publish timestamp:
+                // const publish_time_script = $('script[nonce][type="text/javascript"]:contains("var ct")').html();
+                // const publish_time_match = publish_time_script && publish_time_script.match(/var ct *= *"(\d{10})"/);
+                const publish_time_script = $('script[nonce][type="text/javascript"]:contains("publish_time")').html();
+                const publish_time_match = publish_time_script && publish_time_script.match(/var.*[ ,]n *= *"(\d{10})"/);
+                const publish_timestamp = publish_time_match && publish_time_match[1];
+                if (publish_timestamp) {
+                    item.pubDate = new Date(publish_timestamp * 1000);
```

Comment:
Please use parseDate()
ref: https://github.com/DIYgod/RSSHub/issues/6673
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei
```suggestion
                    item.pubDate = timezone(parseDate(publish_timestamp * 1000), +8);
```

Reply:
I've read these before. A Unix timestamp is always timezone irrelevant, why is a timezone conversion needed? (I've double-checked the timestamp on the page, it does be timezone irrelevant.)

<details>
<summary>And I've done a test:</summary>

publish time: 2022-01-24 21:30:00 UTC+8 (2022-01-24 13:30:00 UTC)
`publish_timestamp`: `1643031000`

<table>
<tr>
	<td>pubDate \ server timezone
	<td>UTC
	<td>UTC+8
<tr>
	<td>new Date(publish_timestamp * 1000)
	<td>Mon, 24 Jan 2022 13:30:00 GMT ⭕
	<td>Mon, 24 Jan 2022 13:30:00 GMT ⭕
<tr>
	<td>parseDate(publish_timestamp * 1000)
	<td>Mon, 24 Jan 2022 13:30:00 GMT ⭕
	<td>Mon, 24 Jan 2022 13:30:00 GMT ⭕
<tr>
	<td>timezone(parseDate(publish_timestamp * 1000), +8)
	<td>Mon, 24 Jan 2022 05:30:00 GMT ❌
	<td>Mon, 24 Jan 2022 13:30:00 GMT ⭕
</table>

</details>

I will change it to `parseDate(publish_timestamp * 1000)` but will not apply a timezone conversion.